pandas/_libs/pd_parser.c

```diff
@@ -0,0 +1,178 @@
+/*
+
+Copyright (c) 2023, PyData Development Team
+All rights reserved.
+
+Distributed under the terms of the BSD Simplified License.
+
+*/
+#define _PANDAS_PARSER_IMPL
+
+#include "pd_parser.h"
+#include "src/parser/io.h"
+
+static int to_double(char *item, double *p_value, char sci, char decimal,
```

Comment:
This and floatify were moved from `parse_helper.h` - seem to more naturally fit in a module than a header